src/devices/Bmx280/Register.cs

```diff
@@ -23,14 +23,25 @@ internal enum Register : byte
         DIG_P8 = 0x9C,
         DIG_P9 = 0x9E,
 
+        // BME280 Only
+        DIG_H1 = 0xA1,
+        DIG_H2 = 0xE1,
+        DIG_H3 = 0xE3,
+        DIG_H4 = 0xE4,
+        DIG_H5 = 0xE5,
+        DIG_H6 = 0xE7,
+
         CHIPID = 0xD0,
         VERSION = 0xD1,
-        SOFTRESET = 0xE0,
+        RESET = 0xE0,
 
         CAL26 = 0xE1,  // R calibration stored in 0xE1-0xF0
 
+        // BME280 Only
+        CTRL_HUM = 0xF2,
+
         STATUS = 0xF3,
-        CONTROL = 0xF4,
+        CTRL_MEAS = 0xF4,
```

Comment:
I suppose this is the name in the DataSheet?

Reply:
Yep.